manifests/common/apps/jupyter/create-jupyter-notebook-quickstart.yaml

```diff
@@ -22,26 +22,26 @@ spec:
       description: |-
         ### Prerequisites
         - You have logged in to Open Data Hub.
-        - You know the names and values that you want to use for environment variables in your notebook server environment, for example, `AWS_SECRET_ACCESS_KEY`.
-        - If you want to work with a very large data set, work with your administrator to proactively increase the storage capacity of your notebook server.
+        - You know the names and values that you want to use for environment variables in your workbench environment, for example, `AWS_SECRET_ACCESS_KEY`.
+        - If you want to work with a very large data set, work with your administrator to proactively increase the storage capacity of your workbench.
         ### Launch a notebook
         1. On the **Enabled applications** page, locate the **Jupyter** card.
         2. Click **Launch application**.
            a) If prompted, select your identity provider.
            b) Enter your credentials and click **Log in** (or equivalent for your identity provider).
            c) If you see an **Access permission needed** message, you are not in the default user group or the default administrator group for Open Data Hub. Contact your administrator so that they can add you to the correct group.
         3. If you have previously launched Jupyter, skip to Step 4.
-           If you have not launched Jupyter previously, start a notebook server.
-           a) Select the **Notebook image** to use for your server.
+           If you have not launched Jupyter previously, start a workbench.
+           a) Select the **Notebook image** to use for your workbench.
            b) If the notebook image contains multiple versions, select the version of the notebook image from the **Versions** section.
-           c) Select the **Container size** for your server.
-        4. (Optional) Select the **Number of GPUs** (graphics processing units) for your server.
+           c) Select the **Container size** for your workbench.
+        4. (Optional) Select the **Number of GPUs** (graphics processing units) for your workbench.
```

Comment:
We call GPUs "Accelerators" elsewhere in the product. If out of scope, please ignore this comment, but if not it may make sense to change it here.